libsrc/pack_utils.c

```diff
@@ -0,0 +1,390 @@
+#ifndef MS_SUA
```

Comment:
We need test code for these C functions. Is there any?

Reply:
No, there are no tests.